pkg/gen/input/workflows/internal/file/create_release.go

```diff
@@ -0,0 +1,198 @@
+package file
+
+import (
+	"github.com/giantswarm/devctl/pkg/gen/input"
+	"github.com/giantswarm/devctl/pkg/gen/input/workflows/internal/params"
+)
+
+func NewCreateReleaseInput(p params.Params) input.Input {
+	i := input.Input{
+		Path:         params.RegenerableFileName(p, "create_release.yaml"),
+		TemplateBody: createReleaseTemplate,
+		TemplateDelims: input.InputTemplateDelims{
+			Left:  "##-ignore-left-##",
+			Right: "##-ignore-right-##",
+		},
```

Comment:
I needed to add this because there is a lot of `${{ ... }}` in the templates and I don't want to change that because we don't really use templating here.